src/extension.ts

```diff
@@ -578,6 +578,51 @@ export function activate(context: vscode.ExtensionContext) {
     )
   );
 
+  subscriptions.push(
+    vscode.commands.registerCommand(
+      "vscode-couchbase.getDocumentMetaData",
+      async (node: DocumentNode) => {
+        const connection = Memory.state.get<IConnection>("activeConnection");
+
+        if (!connection) {
+          return;
+        }
+        try {
+          const viewType = connection.url + "." + node.bucketName + "." + node.scopeName + node.collectionName + "." + node.documentName;
```

Comment:
We're mix string concatenation (`fruit: ` +  myFruit) with string interpolation (`fruit: ${myFruit}`). The preference is to use interpolation, like on the next line when executing a N1QL query.